src/generic/grid.cpp

```diff
@@ -5876,6 +5877,18 @@ void wxGrid::OnKeyDown( wxKeyEvent& event )
                 }
                 break;
 
+            case 'C':
```

Comment:
We still support the "traditional" key combinations in other places, so we should do it here too, could you please add `case WXK_INSERT:` as well?